x-pack/plugins/alerting/server/task_runner/rule_action_helper.test.ts

```diff
@@ -10,7 +10,7 @@ import { RuleAction } from '../types';
 import {
   generateActionHash,
   getSummaryActionsFromTaskState,
-  isSummaryActionOnInterval,
+  isActionOnInterval,
```

Comment:
In general, could we update these unit tests to include possible null or undefined values being passed into the helper functions? I see a lot of accessing possibly undefined values without specifying a default or doing a null check. I know the typescript typings are all good but I always like to add checks to account for unexpected values that might occur in production.

Reply:
done